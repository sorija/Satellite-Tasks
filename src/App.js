import React, { Component } from "react";
// import { Provider } from "react-redux";
import Task from "./Task.js";
import "./styles/App.css";

const mockState = {
  id: 0,
  satName: "Bespin",
  start: 4934.539,
  end: 34943.023,
  status: "new"
};
class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <p>Hello Space!</p>
        </header>

        <Task {...mockState} />
      </div>
    );
  }
}

export default App;
