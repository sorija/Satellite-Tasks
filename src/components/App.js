import React, { Component } from "react";
import { Provider } from "react-redux";
import configureStore from "../configureStore.js";
import RequestsList from "./RequestsList.js";
import AddNewRequest from "./AddNewRequest.js";
import RequestsFilters from "./RequestsFilters.js";
import "../styles/App.css";

const store = configureStore();

class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <div className="App">
          <header className="App-header">
            <p>Hello Space!</p>
          </header>
          <RequestsFilters />
          <AddNewRequest />
          <RequestsList />
        </div>
      </Provider>
    );
  }
}

export default App;
