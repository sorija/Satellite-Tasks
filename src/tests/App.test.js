import React from "react";
import { shallow } from "enzyme";
import App from "../components/App";

test("renders App component", () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
