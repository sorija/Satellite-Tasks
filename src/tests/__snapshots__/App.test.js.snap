// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <div
        className="App space"
      >
        <Connect(RequestsFilters) />
        <Connect(AddNewRequest) />
        <Connect(RequestsList) />
        <figcaption>
          Photo by
           
          <a
            data-href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
            href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
            rel="noopener noreferrer"
            target="_blank"
          >
            NASA
          </a>
           
          on 
          <a
            data-href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
            href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
            rel="noopener noreferrer"
            target="_blank"
          >
            Unsplash
          </a>
        </figcaption>
      </div>,
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(RequestsFilters) />,
          <Connect(AddNewRequest) />,
          <Connect(RequestsList) />,
          <figcaption>
            Photo by
             
            <a
              data-href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
              href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
              rel="noopener noreferrer"
              target="_blank"
            >
              NASA
            </a>
             
            on 
            <a
              data-href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
              href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
              rel="noopener noreferrer"
              target="_blank"
            >
              Unsplash
            </a>
          </figcaption>,
        ],
        "className": "App space",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Photo by",
              " ",
              <a
                data-href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                rel="noopener noreferrer"
                target="_blank"
              >
                NASA
              </a>,
              " ",
              "on ",
              <a
                data-href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                rel="noopener noreferrer"
                target="_blank"
              >
                Unsplash
              </a>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Photo by",
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "NASA",
                "data-href": "https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText",
                "href": "https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": "NASA",
              "type": "a",
            },
            " ",
            "on ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Unsplash",
                "data-href": "https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText",
                "href": "https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": "Unsplash",
              "type": "a",
            },
          ],
          "type": "figcaption",
        },
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="App space"
        >
          <Connect(RequestsFilters) />
          <Connect(AddNewRequest) />
          <Connect(RequestsList) />
          <figcaption>
            Photo by
             
            <a
              data-href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
              href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
              rel="noopener noreferrer"
              target="_blank"
            >
              NASA
            </a>
             
            on 
            <a
              data-href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
              href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
              rel="noopener noreferrer"
              target="_blank"
            >
              Unsplash
            </a>
          </figcaption>
        </div>,
        "store": Object {
          "dispatch": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
          Symbol(observable): [Function],
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Connect(RequestsFilters) />,
            <Connect(AddNewRequest) />,
            <Connect(RequestsList) />,
            <figcaption>
              Photo by
               
              <a
                data-href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                rel="noopener noreferrer"
                target="_blank"
              >
                NASA
              </a>
               
              on 
              <a
                data-href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                rel="noopener noreferrer"
                target="_blank"
              >
                Unsplash
              </a>
            </figcaption>,
          ],
          "className": "App space",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Photo by",
                " ",
                <a
                  data-href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                  href="https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  NASA
                </a>,
                " ",
                "on ",
                <a
                  data-href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                  href="https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  Unsplash
                </a>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "Photo by",
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "NASA",
                  "data-href": "https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText",
                  "href": "https://unsplash.com/photos/rTZW4f02zY8?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "NASA",
                "type": "a",
              },
              " ",
              "on ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Unsplash",
                  "data-href": "https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText",
                  "href": "https://unsplash.com/search/photos/nasa?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText",
                  "rel": "noopener noreferrer",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "Unsplash",
                "type": "a",
              },
            ],
            "type": "figcaption",
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
